vertx-mssql-client/src/main/java/io/vertx/mssqlclient/impl/codec/ExtendedQueryCommandCodec.java

```diff
@@ -476,4 +479,15 @@ private void encodeDecimalParameter(ByteBuf payload, BigDecimal value) {
       payload.writeByte(bytes[i]);
     }
   }
+
+  private void encodeBufferParameter(ByteBuf payload, Buffer value) {
+    payload.writeByte(0x00);
```

Comment:
we should consider minimizing ByteBuf write operations, each write has a cost with respect to bound checks and also often ByteBuf have nested buffers

Reply:
so here instead of 3 write we could have a single write with `0x00_00_00 & MSSQLDataTypeId.BIGBINARYTYPE_ID`

Reply:
Filed https://github.com/eclipse-vertx/vertx-sql-client/issues/965